spacy/displacy/render.py

```diff
@@ -284,6 +293,7 @@ def render_ents(self, text, spans, title):
             label = span["label"]
             start = span["start"]
             end = span["end"]
+            additional_params = span.get("additional_params", {})
```

Comment:
That's a really nice idea! 👍 Maybe we could just call the key `"params"`, to make it a bit shorter?

```suggestion
            additional_params = span.get("params", {})
```